README.md

```diff
@@ -45,9 +38,56 @@ You can check your installed versions by running the following commands from a t
 - `npm --version`
 - `docker --version`
 - `docker-compose --version`
+- `go version`
+
+## Smart Contract Development Lifecycle
+_Please note that all commands contributed by this extension are accesible via the VSCode Command Palette. The majority of the operations described are available from burger menus located on the panel headers, or by right-clicking tree items, in the extension's side bar view._
+
+The expected smart contract development lifecycle follows several broad points, all possible entirely within VSCode using this extension:
+1. Creating and packaging a smart contract
+2. Connecting to an instance of Hyperledger Fabric
+3. Running and debugging a smart contract
+
+### Create and Develop a Fabric smart contract project
+A smart contract project is a directory containing all the relevant contract and metadata files that define a smart contract. Use the `Create Smart Contract Project` command to create a basic smart contract, available in JavaScript, TypeScript, Go or Java. 
+
+### Package a smart contract project
+Once the smart contract has been created and the smart contract transactions have been written, the smart contract project must be packaged before it can be deployed. To package a project you have open in your workspace, run the `Package a Smart Contract Package` command. Packages are listed in the `Smart Contract Packages` panel. The Blockchain output channel lists what files have been packaged during this action. Alternatively run the `Import Package` command to import a pre-existing .cds package to be used within VSCode. 
```

Comment:
"Once the smart contract has been created and the smart contract transactions have been written" what else would you put in the smart contract apart from transaction?

Reply:
Ha I know, but I wanted to explain that the smart contract should (could) be developed before packaging. Perhaps I could just remove this line altogether?